lib/collective-sections.js

```diff
@@ -191,7 +191,6 @@ const getSectionsToRemoveForUser = (collective, isAdmin) => {
   }
   if (features[FEATURES.TRANSACTIONS] !== 'ACTIVE') {
     toRemove.add(Sections.TRANSACTIONS);
-    toRemove.add(Sections.BUDGET);
```

Comment:
This will make sure we display the budget section even if there's no transaction